src/bpm/config/job_config.go

```diff
@@ -130,6 +130,40 @@ func (c *ProcessConfig) Validate(boshRoot string, defaultVolumes []string) error
 	return nil
 }
 
+func (c *ProcessConfig) AddVolumes(volumes []string) error {
+	for _, volume := range volumes {
+		fields := strings.Split(volume, ":")
+
+		if len(fields) > 2 {
+			return fmt.Errorf("invalid volume definition: %s", volume)
```

Comment:
How about we add what a valid volume definition looks like to this error message?